bindgen/templates/RecordTemplate.cs

```diff
@@ -6,6 +6,10 @@
 {%- let (ordered_fields, is_reordered) = rec.fields()|order_fields %}
 
 {%- call cs::docstring(rec, 0) %}
+{%- for field in ordered_fields %}
+{%- call cs::param_docstring(field.name(), field, 0) %}
+{%- endfor %}
+{%- let (ordered_fields, is_reordered) = rec.fields()|order_fields %}
```

Comment:
I had to re-retrieve the fields here because the loop above 'consumed' `ordered_fields`. Is there a better way to do this?